nuxt.config.ts

```diff
@@ -93,7 +93,18 @@ const config: NuxtConfig = {
   buildModules: [
     '@nuxtjs/stylelint-module',
     '@nuxtjs/vuetify',
-    '@nuxt/typescript-build',
+    [
+      '@nuxt/typescript-build',
+      {
+        typeCheck: {
+          async: true,
+          typescript: {
+            enable: true,
+            memoryLimit: 16384,
```

Comment:
@kaizumaki 
私の手元ではmemoryLimitエラーは発生しなかったのですが、ビルドする環境によって発生しうるということであればデフォルトの2048の倍、4096でどうでしょう。16384は大きすぎる印象です。
https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/typescript-reporter/TypeScriptReporterConfiguration.ts#L58

```suggestion
            memoryLimit: 4096,
```

Reply:
ありがとうございます！たしかにそうですね。